bluesky/stack/stack.py

```diff
@@ -780,7 +780,7 @@ def showhelp(cmd=''):
     elif cmd in cmddict:
 
         # Check whether description is available, then show it as well
-        if len(cmddict) <= 3:
+        if len(cmddict[cmd]) <= 4:
```

Comment:
As you add an item to a command entry, the index below is correctly 4 meaning that the length is 5 (or <=4 for no help text).  Also, note it is `cmddict[cmd]` and not `cmddict`